### ISSUE TITLE
Upgrade to 6.5 latest -> Duplicate entry 'Template' for key 'clientName'

### ISSUE THREAD

**Issue by lonesomewalker** [Intermesh/groupoffice#718]
Systemcheck after successful upgrade:

```
Processing GO\Base\Model\Module
Checking 1 GO\Base\Model\Module
Checking 2 GO\Base\Model\Module
Checking 3 GO\Base\Model\Module
Checking 4 GO\Base\Model\Module
Checking 5 GO\Base\Model\Module
Checking 6 GO\Base\Model\Module
Checking 7 GO\Base\Model\Module
Checking 8 GO\Base\Model\Module
Checking 10 GO\Base\Model\Module
Checking 11 GO\Base\Model\Module
Checking 12 GO\Base\Model\Module
Checking 13 GO\Base\Model\Module
Checking 14 GO\Base\Model\Module
Checking 15 GO\Base\Model\Module
Checking 16 GO\Base\Model\Module
Checking 19 GO\Base\Model\Module
Checking 20 GO\Base\Model\Module
Checking 21 GO\Base\Model\Module
Checking 23 GO\Base\Model\Module
Checking 24 GO\Base\Model\Module
Checking 25 GO\Base\Model\Module
Checking 26 GO\Base\Model\Module
Checking 27 GO\Base\Model\Module
Checking 28 GO\Base\Model\Module
Checking 30 GO\Base\Model\Module
Checking 32 GO\Base\Model\Module
Processing GO\Base\Model\AclUsersGroups
Processing GO\Base\Model\SearchCacheRecord
Processing GO\Base\Model\SavedExport
Processing GO\Base\Model\UserGroup
Processing GO\Base\Model\ModelType
Processing GO\Base\Model\Template
Checking 1 GO\Base\Model\Template


SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'Template' for key 'clientName'
```

**Comment by mschering** [comment]
Can you retest this in 6.5.67?

**Comment by mschering** [comment]
Can you send me your core_module and core_entity table?

**Comment by lonesomewalker** [comment]
Sent by email.

**Comment by mschering** [comment]
Thanks, should be fixed in 6.5.70
